web/ui/react-app/src/pages/targets/TargetLabels.tsx

```diff
@@ -15,11 +15,15 @@ export interface TargetLabelsProps {
 
 const formatLabels = (labels: Labels): string[] => Object.keys(labels).map(key => `${key}="${labels[key]}"`);
 
+const sanitizeIDSequence = (scrapePool: string): string => {
+  return scrapePool.replace(/\W/g, (substring: string): string => substring.charCodeAt(0).toString());
```

Comment:
I wonder if I'm being tooooo nitpicky here, but if someone actually does also have a job name that has the number as part of its name, we could produce non-unique IDs here. Can we either properly escape things (using any escape char that is valid in CSS selectors), or alternatively just generate other kinds of unique IDs?

Reply:
That's a good point! I can just escape every character.

Reply:
Actually, I'll look into both options

Reply:
I updated the PR to use `CSS.escape` (with polyfill). Thoughts?